test/WasmSpec/testsuite/js-api/jsapi.js

```diff
@@ -492,6 +492,14 @@ test(() => {
     assert_equals(buf.byteLength, 2 * WasmPage);
     assertThrows(() => mem.grow(1), Error);
     assert_equals(buf, mem.buffer);
+
+    mem = new Memory({initial:0, maximum:1});
+    buf = mem.buffer;
+    assert_equals(buf.byteLength, 0);
+    assert_equals(mem.grow(0), 0);
+    assert_equals(buf !== mem.buffer, true);
+    assert_equals(buf.byteLength, 0);
+    assert_equals(mem.buffer.byteLength, 0);
 }, "'WebAssembly.Memory.prototype.grow' method");

```

Comment:
Would you mind duplicating this test block for an arraybuffer with length 1? There's a difference in what happens in the !ENABLE_FAST_ARRAYBUFFER case between 0-length and nonzero-length arraybuffers, and checking that growing by 0 works there too shouldn't be too much additional code.